### PR TITLE
added loadBalancerIP option to service - Kibana

### PR DESCRIPTION
**Pull request by alexgordonpandera**
- [x] Chart version *not* bumped (the versions are all bumped and released at the same time)
- [x] README.md updated with any new values or changes
- [x] Updated template tests in `${CHART}/tests/*.py` 

Opened to address feature #611

### CODE REVIEW

**Comment by alexgordonpandera** [comment]
I have now signed the CLA

File: kibana/templates/service.yaml [hunk 1]
```diff
@@ -3,10 +3,7 @@ apiVersion: v1
 kind: Service
 metadata:
   name: {{ template "kibana.fullname" . }}
-  labels:
-    app: {{ .Chart.Name }}
-    release: {{ .Release.Name | quote }}
-    heritage: {{ .Release.Service }}
+  labels: {{ include "kibana.labels" . | nindent 4 }}
```
Comment (jmlrt) [inline kibana/templates/service.yaml:6]: This change has been implemented in https://github.com/elastic/helm-charts/pull/555. Can you merge master in your branch?

File: kibana/tests/kibana_test.py [hunk 2]
```diff
@@ -89,6 +89,19 @@ def test_overriding_the_port():
     assert r["service"][name]["spec"]["ports"][0]["targetPort"] == 5602
 
 
+def test_adding_env_from():
```
Comment (jmlrt) [inline kibana/tests/kibana_test.py:92]: implemented in  #569

**Comment by jmlrt** [comment]
jenkins test this please

**Comment by jmlrt** [comment]
cla/check

File: kibana/templates/service.yaml (hunk 1 above)
Comment (alexgordonpandera) [reply kibana/templates/service.yaml:6]: Merged

File: kibana/tests/kibana_test.py (hunk 2 above)
Comment (alexgordonpandera) [reply kibana/tests/kibana_test.py:92]: Merged

**Comment by alexgordonpandera** [comment]
As for the README. You do not have the same structure in your README for Kibana. It does not makes sense to copy what was in elasticsearch unless you are going to add all of the services.xxx sub-values to the README. Currently it links to the values.yaml, which has been updated to display how to use loadBalancerIP.

I notice it failed the linter in the CI/CD pipeline. What errors came from the linter so I can fix them.

**Comment by jmlrt** [comment]
jenkins test this please

**Comment by jmlrt** [comment]
> I notice it failed the linter in the CI/CD pipeline. What errors came from the linter so I can fix them.

https://devops-ci.elastic.co/job/elastic+helm-charts+pull-request+lint-python/285/label=docker&&virtual&&ubuntu-18.04/consoleText:
```
+ make lint-python
black --diff --check --exclude='ve/|venv/' .
--- kibana/tests/kibana_test.py	2020-06-17 21:15:10.466160 +0000
would reformat kibana/tests/kibana_test.py
+++ kibana/tests/kibana_test.py	2020-06-17 21:16:00.210955 +0000
@@ -465,10 +465,11 @@
 
     r = helm_template(config)
 
     assert r["service"][name]["spec"]["ports"][0]["nodePort"] == 30001
 
+
 def test_adding_a_loadBalancerIP():
     config = ""
 
     r = helm_template(config)
 
@@ -480,10 +481,11 @@
     """
Oh no! 💥 💔 💥
1 file would be reformatted, 8 files would be left unchanged.
 
     r = helm_template(config)
 
     assert r["service"][name]["spec"]["loadBalancerIP"] == "12.4.19.81"
+
 
 def test_override_the_serverHost():
     config = """
     serverHost: "localhost"
     """
make: *** [lint-python] Error 1
```

https://devops-ci.elastic.co/job/elastic+helm-charts+pull-request+template-testing/664/CHART=kibana,label=docker&&virtual/consoleText:

```
helm lint --strict ./
==> Linting ./
Error: 1 chart(s) linted, 1 chart(s) failed
[ERROR] templates/: render error in "kibana/templates/service.yaml": template: kibana/templates/service.yaml:6:13: executing "kibana/templates/service.yaml" at <include "kibana.labels" .>: error calling include: template: no template "kibana.labels" associated with template "gotpl"

make: *** [../helpers/common.mk:27: lint] Error 1
```

**Comment by jmlrt** [comment]
Your commit still contains some changes which are not relevant for this PR and were already applied in other PR:

Could reset your branch and reapply your changes related to `loadBalancerIP`:
- https://github.com/elastic/helm-charts/pull/666/files#diff-25bc1bad9f2943882d2368d37fe703a3R31-R33
- https://github.com/elastic/helm-charts/pull/666/files#diff-0cb7035b163a9e0936cca71dafa124aaR109
- https://github.com/elastic/helm-charts/pull/666/files#diff-83315ff6e6f0d926f78661eebc687d0cR470-R484

```bash
# reset branch
git reset origin/master --hard

# apply your changes to kibana/templates/service.yaml kibana/tests/kibana_test.py kibana/values.yaml
# ...

# recreate commit
git add kibana/templates/service.yaml kibana/tests/kibana_test.py kibana/values.yaml
git commit -m 'added loadBalancerIP option to service - Kibana '

# push force
git push --force
```

**Comment by alexgordonpandera** [comment]
Alright it should be good to go

**Comment by jmlrt** [comment]
That's really strange, your new commit https://github.com/elastic/helm-charts/pull/666/commits/444a2cc618a921d42ce5278161d65b8ddbceb823 still contain the changes from other PRs.

I'm sorry for this but maybe it would be easier to recreate a new branch from master and a new PR to have only the changes require.

I don't see another solution.

**Comment by debojitkakoti** [comment]
Is it merged in different PR?

**Comment by jmlrt** [comment]
Replaced by #726

**Pull request closed without merging**